razorpay-payments.php

```diff
@@ -8,10 +8,14 @@
 Author URI: https://razorpay.com
 */
 
+require_once __DIR__.'/includes/razorpay-webhook.php';
+
 require_once __DIR__.'/razorpay-sdk/Razorpay.php';
 use Razorpay\Api\Api;
 
 add_action('plugins_loaded', 'woocommerce_razorpay_init', 0);
+add_action('admin_post_nopriv_rzp_webhook', 'razorpay_webhook_init'); // second - admin_post_nopriv
```

Comment:
@captn3m0 if you want to have both logged in and not logged in users submitting, you have to add both actions!

https://codex.wordpress.org/Plugin_API/Action_Reference/admin_post_(action)